### ISSUE TITLE
Import of bigip_ltm_virtual_server crashes terraform

### ISSUE THREAD

**Issue by yunal** [F5Networks/terraform-provider-bigip#729]
### Environment

 * TMOS/Bigip Version: 15.1.8 Build 0.0.7
 * Terraform Version: v1.3.5
 * Terraform bigip provider Version: 1.16.0

### Summary

plugin crashes while importing bigip_ltm_virtual_server resource

### Steps To Reproduce

1. Provide terraform resource config which you are facing trouble along with the output of it.

Made skeleton resource to allow import:
```
resource "bigip_ltm_virtual_server" "<sensitive>" {
  name       = "/Common/<sensitive>"
}
```

Issued import command: `terraform import bigip_ltm_virtual_server.<sensitive> /Common/<sensitive>`

3. To get to know more about the issue, provide terraform debug logs

```
2022-11-24T08:56:29.663+0100 [INFO]  Terraform version: 1.3.5
2022-11-24T08:56:29.663+0100 [DEBUG] using github.com/hashicorp/go-tfe v1.9.0
2022-11-24T08:56:29.663+0100 [DEBUG] using github.com/hashicorp/hcl/v2 v2.15.0
2022-11-24T08:56:29.663+0100 [DEBUG] using github.com/hashicorp/terraform-config-inspect v0.0.0-20210209133302-4fd17a0faac2
2022-11-24T08:56:29.663+0100 [DEBUG] using github.com/hashicorp/terraform-svchost v0.0.0-20200729002733-f050f53b9734
2022-11-24T08:56:29.663+0100 [DEBUG] using github.com/zclconf/go-cty v1.12.1
2022-11-24T08:56:29.663+0100 [INFO]  Go runtime version: go1.19.3
2022-11-24T08:56:29.663+0100 [INFO]  CLI args: []string{"terraform", "import", "bigip_ltm_virtual_server.<sensitive>", "/Common/<sensitive>"}
2022-11-24T08:56:29.663+0100 [DEBUG] Attempting to open CLI config file: /home/niclangf/.terraformrc
2022-11-24T08:56:29.663+0100 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2022-11-24T08:56:29.663+0100 [DEBUG] ignoring non-existing provider search directory terraform.d/plugins
2022-11-24T08:56:29.663+0100 [DEBUG] ignoring non-existing provider search directory /home/niclangf/.terraform.d/plugins
2022-11-24T08:56:29.663+0100 [DEBUG] ignoring non-existing provider search directory /home/niclangf/.local/share/terraform/plugins
2022-11-24T08:56:29.663+0100 [DEBUG] ignoring non-existing provider search directory /usr/share/plasma/terraform/plugins
2022-11-24T08:56:29.663+0100 [DEBUG] ignoring non-existing provider search directory /home/niclangf/.local/share/flatpak/exports/share/terraform/plugins
2022-11-24T08:56:29.663+0100 [DEBUG] ignoring non-existing provider search directory /var/lib/flatpak/exports/share/terraform/plugins
2022-11-24T08:56:29.663+0100 [DEBUG] ignoring non-existing provider search directory /usr/local/share/terraform/plugins
2022-11-24T08:56:29.663+0100 [DEBUG] ignoring non-existing provider search directory /usr/share/terraform/plugins
2022-11-24T08:56:29.663+0100 [DEBUG] ignoring non-existing provider search directory /var/lib/snapd/desktop/terraform/plugins
2022-11-24T08:56:29.663+0100 [INFO]  CLI command args: []string{"import", "bigip_ltm_virtual_server.<sensitive>", "/Common/<sensitive>"}
2022-11-24T08:56:29.665+0100 [DEBUG] New state was assigned lineage "fc0cf905-e52d-cf58-c77a-2b966b7abbbf"
2022-11-24T08:56:29.705+0100 [DEBUG] checking for provisioner in "."
2022-11-24T08:56:29.708+0100 [DEBUG] checking for provisioner in "/usr/bin"
2022-11-24T08:56:29.711+0100 [DEBUG] Building and walking import graph
2022-11-24T08:56:29.711+0100 [DEBUG] ProviderTransformer: "bigip_ltm_virtual_server.api_stream-k8s_vgnett_no (expand)" (*terraform.nodeExpandPlannableResource) needs provider["registry.terraform.io/f5networks/bigip"]
2022-11-24T08:56:29.711+0100 [DEBUG] ProviderTransformer: "bigip_ltm_virtual_server.<sensitive> (expand)" (*terraform.nodeExpandPlannableResource) needs provider["registry.terraform.io/f5networks/bigip"]
2022-11-24T08:56:29.711+0100 [DEBUG] ProviderTransformer: "bigip_ltm_pool_attachment.<sensitive>-node-02_444 (expand)" (*terraform.nodeExpandPlannableResource) needs provider["registry.terraform.io/f5networks/bigip"]
2022-11-24T08:56:29.711+0100 [DEBUG] ProviderTransformer: "bigip_ltm_node.stream-k8s-master-02_int_vgnett_no (expand)" (*terraform.nodeExpandPlannableResource) needs provider["registry.terraform.io/f5networks/bigip"]
2022-11-24T08:56:29.711+0100 [DEBUG] ProviderTransformer: "bigip_ltm_pool.<sensitive> (expand)" (*terraform.nodeExpandPlannableResource) needs provider["registry.terraform.io/f5networks/bigip"]
2022-11-24T08:56:29.711+0100 [DEBUG] ProviderTransformer: "bigip_ltm_pool.k8s-stream-api (expand)" (*terraform.nodeExpandPlannableResource) needs provider["registry.terraform.io/f5networks/bigip"]
2022-11-24T08:56:29.711+0100 [DEBUG] ProviderTransformer: "bigip_ltm_node.<sensitive>-node-02 (expand)" (*terraform.nodeExpandPlannableResource) needs provider["registry.terraform.io/f5networks/bigip"]
2022-11-24T08:56:29.711+0100 [DEBUG] ProviderTransformer: "bigip_ltm_pool_attachment.k8s-stream-api_common_stream-k8s-master-02_int_vgnett_no_6443 (expand)" (*terraform.nodeExpandPlannableResource) needs provider["registry.terraform.io/f5networks/bigip"]
2022-11-24T08:56:29.711+0100 [DEBUG] ProviderTransformer: "bigip_ltm_node.stream-k8s-master-01_int_vgnett_no (expand)" (*terraform.nodeExpandPlannableResource) needs provider["registry.terraform.io/f5networks/bigip"]
2022-11-24T08:56:29.711+0100 [DEBUG] ProviderTransformer: "bigip_ltm_node.stream-k8s-master-03_int_vgnett_no (expand)" (*terraform.nodeExpandPlannableResource) needs provider["registry.terraform.io/f5networks/bigip"]
2022-11-24T08:56:29.711+0100 [DEBUG] ProviderTransformer: "bigip_ltm_pool_attachment.k8s-stream-api_common_stream-k8s-master-01_int_vgnett_no_6443 (expand)" (*terraform.nodeExpandPlannableResource) needs provider["registry.terraform.io/f5networks/bigip"]
2022-11-24T08:56:29.711+0100 [DEBUG] ProviderTransformer: "bigip_ltm_pool_attachment.k8s-stream-api_common_stream-k8s-master-03_int_vgnett_no_6443 (expand)" (*terraform.nodeExpandPlannableResource) needs provider["registry.terraform.io/f5networks/bigip"]
2022-11-24T08:56:29.711+0100 [DEBUG] created provider logger: level=debug
2022-11-24T08:56:29.711+0100 [INFO]  provider: configuring client automatic mTLS
2022-11-24T08:56:29.732+0100 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/f5networks/bigip/1.16.0/linux_amd64/terraform-provider-bigip_v1.16.0 args=[.terraform/providers/registry.terraform.io/f5networks/bigip/1.16.0/linux_amd64/terraform-provider-bigip_v1.16.0]
2022-11-24T08:56:29.732+0100 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/f5networks/bigip/1.16.0/linux_amd64/terraform-provider-bigip_v1.16.0 pid=20930
2022-11-24T08:56:29.732+0100 [DEBUG] provider: waiting for RPC address: path=.terraform/providers/registry.terraform.io/f5networks/bigip/1.16.0/linux_amd64/terraform-provider-bigip_v1.16.0
2022-11-24T08:56:29.738+0100 [INFO]  provider.terraform-provider-bigip_v1.16.0: configuring server automatic mTLS: timestamp=2022-11-24T08:56:29.738+0100
2022-11-24T08:56:29.754+0100 [DEBUG] provider: using plugin: version=5
2022-11-24T08:56:29.754+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: plugin address: address=/tmp/plugin3345191434 network=unix timestamp=2022-11-24T08:56:29.754+0100
2022-11-24T08:56:29.767+0100 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unimplemented desc = unknown service plugin.GRPCStdio"
2022-11-24T08:56:29.771+0100 [DEBUG] No provider meta schema returned
2022-11-24T08:56:29.775+0100 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/f5networks/bigip/1.16.0/linux_amd64/terraform-provider-bigip_v1.16.0 pid=20930
2022-11-24T08:56:29.775+0100 [DEBUG] provider: plugin exited
2022-11-24T08:56:29.775+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_virtual_server.api_stream-k8s_vgnett_no (expand)" references: []
2022-11-24T08:56:29.775+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_virtual_server.<sensitive> (expand)" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_pool_attachment.k8s-stream-api_common_stream-k8s-master-02_int_vgnett_no_6443 (expand)" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_node.stream-k8s-master-01_int_vgnett_no (expand)" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "var.hostname" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "var.username" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "provider[\"registry.terraform.io/f5networks/bigip\"]" references: [var.username var.password var.hostname]
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_node.stream-k8s-master-03_int_vgnett_no (expand)" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_pool_attachment.k8s-stream-api_common_stream-k8s-master-01_int_vgnett_no_6443 (expand)" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_pool_attachment.k8s-stream-api_common_stream-k8s-master-03_int_vgnett_no_6443 (expand)" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_pool_attachment.<sensitive>-node-02_444 (expand)" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_pool.k8s-stream-api (expand)" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "var.password" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_node.stream-k8s-master-02_int_vgnett_no (expand)" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_node.<sensitive>-node-02 (expand)" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_pool.<sensitive> (expand)" references: []
2022-11-24T08:56:29.776+0100 [DEBUG] Starting graph walk: walkImport
2022-11-24T08:56:29.777+0100 [DEBUG] created provider logger: level=debug
2022-11-24T08:56:29.777+0100 [INFO]  provider: configuring client automatic mTLS
2022-11-24T08:56:29.785+0100 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/f5networks/bigip/1.16.0/linux_amd64/terraform-provider-bigip_v1.16.0 args=[.terraform/providers/registry.terraform.io/f5networks/bigip/1.16.0/linux_amd64/terraform-provider-bigip_v1.16.0]
2022-11-24T08:56:29.785+0100 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/f5networks/bigip/1.16.0/linux_amd64/terraform-provider-bigip_v1.16.0 pid=20946
2022-11-24T08:56:29.785+0100 [DEBUG] provider: waiting for RPC address: path=.terraform/providers/registry.terraform.io/f5networks/bigip/1.16.0/linux_amd64/terraform-provider-bigip_v1.16.0
2022-11-24T08:56:29.791+0100 [INFO]  provider.terraform-provider-bigip_v1.16.0: configuring server automatic mTLS: timestamp=2022-11-24T08:56:29.791+0100
2022-11-24T08:56:29.809+0100 [DEBUG] provider: using plugin: version=5
2022-11-24T08:56:29.809+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: plugin address: network=unix address=/tmp/plugin1838278970 timestamp=2022-11-24T08:56:29.809+0100
2022-11-24T08:56:29.822+0100 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unimplemented desc = unknown service plugin.GRPCStdio"
2022-11-24T08:56:29.826+0100 [DEBUG] No provider meta schema returned
2022-11-24T08:56:29.835+0100 [WARN]  ValidateProviderConfig from "provider[\"registry.terraform.io/f5networks/bigip\"]" changed the config value, but that value is unused
2022-11-24T08:56:29.835+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: 2022/11/24 08:56:29 [INFO] Initializing BigIP connection
2022-11-24T08:56:30.235+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_node.stream-k8s-master-03_int_vgnett_no" references: []
2022-11-24T08:56:30.235+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_pool.k8s-stream-api" references: []
2022-11-24T08:56:30.236+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_virtual_server.api_stream-k8s_vgnett_no" references: []
2022-11-24T08:56:30.237+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_pool_attachment.k8s-stream-api_common_stream-k8s-master-02_int_vgnett_no_6443" references: []
2022-11-24T08:56:30.237+0100 [DEBUG] Resource instance state not found for node "bigip_ltm_pool_attachment.<sensitive>-node-02_444", instance bigip_ltm_pool_attachment.<sensitive>-node-02_444
2022-11-24T08:56:30.238+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_pool_attachment.<sensitive>-node-02_444" references: []
2022-11-24T08:56:30.238+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_pool_attachment.k8s-stream-api_common_stream-k8s-master-03_int_vgnett_no_6443" references: []
2022-11-24T08:56:30.238+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_node.stream-k8s-master-01_int_vgnett_no" references: []
2022-11-24T08:56:30.238+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_node.stream-k8s-master-02_int_vgnett_no" references: []
2022-11-24T08:56:30.239+0100 [DEBUG] Resource instance state not found for node "bigip_ltm_node.<sensitive>-node-02", instance bigip_ltm_node.<sensitive>-node-02
2022-11-24T08:56:30.239+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_node.<sensitive>-node-02" references: []
2022-11-24T08:56:30.239+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_virtual_server.<sensitive> (import id \"/Common/<sensitive>\")" references: []
bigip_ltm_virtual_server.<sensitive>: Importing from ID "/Common/<sensitive>"...
2022-11-24T08:56:30.239+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_pool_attachment.k8s-stream-api_common_stream-k8s-master-01_int_vgnett_no_6443" references: []
2022-11-24T08:56:30.239+0100 [DEBUG] Resource instance state not found for node "bigip_ltm_pool.<sensitive>", instance bigip_ltm_pool.<sensitive>
2022-11-24T08:56:30.239+0100 [DEBUG] ReferenceTransformer: "bigip_ltm_pool.<sensitive>" references: []
bigip_ltm_virtual_server.<sensitive>: Import prepared!
  Prepared bigip_ltm_virtual_server for import
bigip_ltm_virtual_server.<sensitive>: Refreshing state... [id=/Common/<sensitive>]
2022-11-24T08:56:30.242+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: 2022/11/24 08:56:30 [INFO] Fetching virtual server /Common/<sensitive>
2022-11-24T08:56:31.082+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: 2022/11/24 08:56:31 [INFO] Fetching virtual server /Common/<sensitive>
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: panic: runtime error: index out of range [2] with length 0
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: 
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: goroutine 52 [running]:
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: github.com/F5Networks/terraform-provider-bigip/bigip.resourceBigipLtmVirtualServerRead(0xc000860150, {0xf27a20?, 0xc00049a9a0})
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: github.com/F5Networks/terraform-provider-bigip/bigip/resource_bigip_ltm_virtual_server.go:323 +0x2045
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc000053c00, 0xc000746410, {0xf27a20, 0xc00049a9a0})
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: github.com/hashicorp/terraform-plugin-sdk@v1.1.0/helper/schema/resource.go:455 +0x1aa
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ReadResource(0xc0000121e8, {0xc000746230?, 0x59ab66?}, 0xc000746230)
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: github.com/hashicorp/terraform-plugin-sdk@v1.1.0/internal/helper/plugin/grpc_provider.go:525 +0x365
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ReadResource_Handler({0xeee660?, 0xc0000121e8}, {0x10f04b0, 0xc000c0e7b0}, 0xc0005002a0, 0x0)
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: github.com/hashicorp/terraform-plugin-sdk@v1.1.0/internal/tfplugin5/tfplugin5.pb.go:3153 +0x170
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: google.golang.org/grpc.(*Server).processUnaryRPC(0xc0001ae480, {0x10f4520, 0xc0001ae600}, 0xc000992100, 0xc000710060, 0x16b6370, 0x0)
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: google.golang.org/grpc@v1.23.0/server.go:995 +0xe1e
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: google.golang.org/grpc.(*Server).handleStream(0xc0001ae480, {0x10f4520, 0xc0001ae600}, 0xc000992100, 0x0)
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: google.golang.org/grpc@v1.23.0/server.go:1275 +0xa16
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: google.golang.org/grpc.(*Server).serveStreams.func1.1()
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: google.golang.org/grpc@v1.23.0/server.go:710 +0x98
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: created by google.golang.org/grpc.(*Server).serveStreams.func1
2022-11-24T08:56:31.316+0100 [DEBUG] provider.terraform-provider-bigip_v1.16.0: google.golang.org/grpc@v1.23.0/server.go:708 +0xea
2022-11-24T08:56:31.320+0100 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/f5networks/bigip/1.16.0/linux_amd64/terraform-provider-bigip_v1.16.0 pid=20946 error="exit status 2"
2022-11-24T08:56:31.320+0100 [ERROR] plugin.(*GRPCProvider).ReadResource: error="rpc error: code = Unavailable desc = error reading from server: EOF"
2022-11-24T08:56:31.320+0100 [ERROR] vertex "import bigip_ltm_virtual_server.<sensitive> result" error: Plugin did not respond
2022-11-24T08:56:31.320+0100 [ERROR] vertex "bigip_ltm_virtual_server.<sensitive> (import id \"/Common/<sensitive>\")" error: Plugin did not respond
2022-11-24T08:56:31.320+0100 [ERROR] vertex "bigip_ltm_virtual_server.<sensitive> (expand)" error: Plugin did not respond
╷
│ Error: Plugin did not respond
│ 
│ The plugin encountered an error, and failed to respond to the
│ plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more
│ details.
╵


Stack trace from the terraform-provider-bigip_v1.16.0 plugin:

panic: runtime error: index out of range [2] with length 0

goroutine 52 [running]:
github.com/F5Networks/terraform-provider-bigip/bigip.resourceBigipLtmVirtualServerRead(0xc000860150, {0xf27a20?, 0xc00049a9a0})
        github.com/F5Networks/terraform-provider-bigip/bigip/resource_bigip_ltm_virtual_server.go:323 +0x2045
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc000053c00, 0xc000746410, {0xf27a20, 0xc00049a9a0})
        github.com/hashicorp/terraform-plugin-sdk@v1.1.0/helper/schema/resource.go:455 +0x1aa
github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ReadResource(0xc0000121e8, {0xc000746230?, 0x59ab66?}, 0xc000746230)
        github.com/hashicorp/terraform-plugin-sdk@v1.1.0/internal/helper/plugin/grpc_provider.go:525 +0x365
github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ReadResource_Handler({0xeee660?, 0xc0000121e8}, {0x10f04b0, 0xc000c0e7b0}, 0xc0005002a0, 0x0)
        github.com/hashicorp/terraform-plugin-sdk@v1.1.0/internal/tfplugin5/tfplugin5.pb.go:3153 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0001ae480, {0x10f4520, 0xc0001ae600}, 0xc000992100, 0xc000710060, 0x16b6370, 0x0)
        google.golang.org/grpc@v1.23.0/server.go:995 +0xe1e
google.golang.org/grpc.(*Server).handleStream(0xc0001ae480, {0x10f4520, 0xc0001ae600}, 0xc000992100, 0x0)
        google.golang.org/grpc@v1.23.0/server.go:1275 +0xa16
google.golang.org/grpc.(*Server).serveStreams.func1.1()
        google.golang.org/grpc@v1.23.0/server.go:710 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.23.0/server.go:708 +0xea

Error: The terraform-provider-bigip_v1.16.0 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.

2022-11-24T08:56:31.321+0100 [DEBUG] provider: plugin exited
```

### Expected Behavior

Successful import.

### Actual Behavior

Crash. See log.  This happens with at least two VIPs. Otherwise all the VIPs get imported with no issue.

I can provide VIP definition etc through more secure f5 support channels.

**Comment by RavinderReddyF5** [comment]
Hi @yunal, may i know why it is used as `<sensitive>` in resource config, as far as i remember The resource type and name together serve as an identifier for a given resource and so must be unique within a module

https://developer.hashicorp.com/terraform/language/resources/syntax

**Comment by yunal** [comment]
You are right indeed, and also "<" and ">" is not allowed in resource names.

I've censored the config because this is actual configuration from our BigIP and the names would give away technologies and products we use which we might not like that random people on the internet know.  Also I suspect that the developer to debug this needs access to our actual configuration.  Therefore I've also entered a bug with BigIP referencing this github ticket and given them our complete BigIP config so they can test this.

**Comment by RavinderReddyF5** [comment]
@yunal , i dont see any issue import with basic virtual server, can you please create support case with config and debug logs if need further help

**Comment by yunal** [comment]
Yes, asI said I have importet many VIPs without issue. And as I said I've opened a case and shared config with f5.

**Comment by yunal** [comment]
@RavinderReddyF5 - sorry I've been argumentative and that was uncalled for.

**Comment by niclan** [comment]
Here is my script to generate terraform resources from a running bigip https://github.com/schibsted/bigip-to-terraform

**Comment by eos500n2000** [comment]
@RavinderReddyF5  I created an escalation case 00323427.

According to my lab test,
VS with "traffic-matching-criteria" parameter can not be imported by terraform.

Set "traffic-matching-criteria" to none, import works as expected.


ltm virtual /Common/VS-name {

ip-protocol tcp

pool /Common/pool-name

profiles {

/Common/tcp { }

}

serverssl-use-sni disabled

traffic-matching-criteria /Common/traffic-matching-criteria-name       ###### with this set to not "NONE"

translate-address enabled

translate-port enabled

}

**Comment by KrithikaChidambaram** [comment]
Hi, we are tracking this in our backlog, internal ID: INFRAANO-993.

**Comment by niclan** [comment]
Our F5 contact for this issue reverted today:
> We have raised internal Product development request and status is set to done, which should most probably available in next release i.e., Feb 2023

So I'm awaiting the February release.

**Comment by RavinderReddyF5** [comment]
HI @yunal @niclan fixed in v1.16.2
